test/reuse/util/file/getExcelData.spec.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+
+describe("file - getExcelData - Retrieve xls file data", function () {
+  let value;
+
+  it("Execution", async function () {
+    const xlsPath = path.resolve(__dirname, "./testFiles");
+    value = await util.file.getExcelData(xlsPath, "test.xls", 1);
+  });
+
+  it("Verification", async function () {
+    common.assertion.expectDefined(value);
```

Comment:
instead of the generic assertion, you can check for the exact output or assert parts of the output

Reply:
I have added specific assertion to test instead of generic